### ISSUE TITLE
Game crash when trying to deploy.

### ISSUE THREAD

**Issue by 0x9conc1** [CleverRaven/Cataclysm-DDA#45246]
When i have tried to deploy (expand) the game window it crashed. It was when i was at the main menu of minimized game; i pressed alt + tab to expand, the shortcut was highlighted (the game window became active this means), but the game did not expand in 5 seconds, clicked again to defocus the active window and activate it again, 2-3 times and crash. In the settings, the game starts in full-screen mode.
[crash&debug.zip](https://github.com/CleverRaven/Cataclysm-DDA/files/5490212/crash.debug.zip)

Other information:
 - OS: Windows 7 64bit
 - Game Version: 0.E-2
 - Ingame language: System (Russian)
# Update
In short, to check the bug: play in full-screen mode and from time to time minimize, expand the game.

**Comment by AnonimusC** [comment]
[1] in the main menu, went to settings, opened the graphics tab to change the game screen resolution, collapsed, but could not expand - error.
[2] when expanded - black background on the full screen, collapsed, later a message popped up and then another one. similarly, in [1]
![image](https://user-images.githubusercontent.com/73962707/99436194-333a0780-2922-11eb-9052-968fd9f37bc1.png)
![image](https://user-images.githubusercontent.com/73962707/99436209-3b924280-2922-11eb-98da-bac94d814841.png)

**Comment by Night-Pryanik** [comment]
Closing as stale, since stalebot can't do this by itself.
